### DISCUSSION TITLE
Switch to chose quality between original img and  preview-img

### DISCUSSION THREAD

**Discussion by fabian1512** [damongolding/immich-kiosk#171] [Feature request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

because I want to save memory due to my large number of images for Immich, I have set the settings for the preview images to webp. My Lan/WLan has sufficient bandwidth so that I can always use the original images.
However, Kiosk seems to use the preview images.
It would be great to be able to choose the quality between preview and original in the config.

**Comment by damongolding** [comment]
I can add this. Though I might have to add a caveat that it could cause kiosk to use more ram/cpu resources.

**Comment by damongolding** [comment]
added in 0.13.0
